macros.in

```diff
@@ -1073,7 +1073,8 @@ package or when debugging this package.\
 
 #------------------------------------------------------------------------------
 # The "make" analogue, hiding the _smp_mflags magic from specs
-%make_build %{__make} %{_make_output_sync} %{?_smp_mflags} V=1 VERBOSE=1
+%_make_verbose V=1 VERBOSE=1
+%make_build %{__make} %{_make_output_sync} %{?_smp_mflags} %{_make_verbose}
```

Comment:
`%{_make_verbose}` -> `%{?_make_verbose}`

Also, can you put the `%_make_verbose` macro in its own "section" like `%_make_output_sync` was?

Reply:
If I did everything correctly, I believe I've done what you requested.